glossary/glossary.md

```diff
@@ -72,7 +107,7 @@ SIEC - Server Idle Energy Coefficiency
 
 
 
-## Ressources
-* [1] Glossary of Climate Change - https://www.wikiwand.com/en/Glossary_of_climate_change
-* [2] Climate Change Vocabulary - https://www.health.state.mn.us/communities/environment/climate/docs/film/vocab_list.pdf
+## Resources
 
+* [1] Glossary of Climate Change - https://www.wikiwand.com/en/Glossary_of_climate_change
+* [2] Climate Change Vocabulary - https://www.health.state.mn.us/communities/environment/climate/docs/film/vocab_list.pdf
```

Comment:
Resources 1 and 2 are not linked to anything on this page.

Reply:
I think it's more of a hint to show where one can read more or where we took the terms from initially. Maybe when we add "Additional" Resources we will be fine, and everything will be clear 🤔